src/GitHubParserHook.php

```diff
@@ -43,14 +53,58 @@ private function setFields( ProcessingResult $result ) {
 		$this->fileName = $params['file']->getValue();
 		$this->repoName = $params['repo']->getValue();
 		$this->branchName = $params['branch']->getValue();
+
+		foreach ( self::$syntaxHighlightFields as $val ) {
+			if ( isset( $params[$val] ) ) {
+				$this->$val = $this->cleanField( $params[$val]->getValue() );
+			}
+			else {
+				$this->$val = null;
+			}
+		}
 	}
 
-	private function getRenderedContent() {
+	private function cleanField( $val ) {
+		if ( $val !== null ) {
+			$val = trim( $val, "'\"" );
+		}
+		return $val;
+	}
+
+	private function getRenderedContent(Parser $parser) {
 		$content = $this->getFileContent();
 
 		if ( $this->isMarkdownFile() ) {
 			$content = $this->renderAsMarkdown( $content );
 		}
+		else if ($this->lang !== "") {
+			if ( ExtensionRegistry::getInstance()->isLoaded( 'SyntaxHighlight' ) ) {
```

Comment:
ExtensionRegistry? The tests are failing cause of this reference

https://travis-ci.org/JeroenDeDauw/GitHub/jobs/215355374